apps/nextjs-website/src/components/molecules/QuestionsAndAnswers/QuestionsAndAnswers.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Compress, Expand, ExpandMore } from '@mui/icons-material';
+import { useTranslations } from 'next-intl';
+import { QuestionsAndAnswers } from '@/lib/types/webinar';
+import BlocksRendererClient from '../BlocksRendererClient/BlocksRendererClient';
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+
+const MIN_QUESTIONS_TO_SHOW = 5;
+const MAX_QUESTIONS_TO_SHOW = 10;
+
+export type QuestionsAndAnswersProps = {
+  readonly questions: QuestionsAndAnswers[];
+};
+
+const QuestionsAndAnswersComponent = ({
```

Comment:
i will change `QuestionsAndAnswersComponent` into `QuestionsAndAnswers` like other components

Reply:
I named the component like this because of the `QuestionsAndAnswers` interface that would create conflicts